modules/profiles/handle_periodic_operations.go

```diff
@@ -0,0 +1,22 @@
+package profiles
+
+import (
+	"github.com/forbole/bdjuno/v2/modules/utils"
+
+	"github.com/go-co-op/gocron"
+	"github.com/rs/zerolog/log"
+)
+
+// RegisterPeriodicOperations implements modules.PeriodicOperationsModule
+func (m *Module) RegisterPeriodicOperations(scheduler *gocron.Scheduler) error {
```

Comment:
Do we need to get this value everyday? Can we store the params when we run ```bdjuno parse-genesis``` and refresh those values when the proposal has passed?

Reply:
that's true! I added it into gov module :D